src/Monitor/Monitor/Utilities.cs

```diff
@@ -84,5 +84,11 @@ public static string ReadFileContent(string path)
             using (TextReader reader = new StreamReader(path))
                 return reader.ReadToEnd();
         }
+
+        public static bool IsGuid(string str)
+        {
+            Guid g;
+            return Guid.TryParse(str, out g);
```

Comment:
Syntax sugar:
```suggestion
            return Guid.TryParse(str, out _);
```
see [Discards](https://docs.microsoft.com/en-us/dotnet/csharp/discards)